<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>A Channel</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data receiving area</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>77</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="button_start_a">
       <property name="text">
        <string>Start A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_stop_a">
       <property name="text">
        <string>Stop A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_clear_a">
       <property name="text">
        <string>Clear A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_save_a">
       <property name="text">
        <string>Save A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textedit_a_area">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>561</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>B Channel</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data receiving area</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>77</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="button_start_b">
       <property name="text">
        <string>Start B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_stop_b">
       <property name="text">
        <string>Stop B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_clear_b">
       <property name="text">
        <string>Clear B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_save_b">
       <property name="text">
        <string>Save B</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textedit_b_area">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Multi channel acquisition system.</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>181</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>208</y>
      <width>126</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>UAR选择：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobutton_a_uart">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobutton_b_uart">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>A采样率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>B采样率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>A峰值：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>B峰值：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineedit_a_samplerate">
         <property name="text">
          <string>16000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineedit_b_samplerate">
         <property name="text">
          <string>16000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineedit_a_peakvalue">
         <property name="text">
          <string>3.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineedit_b_peakvalue">
         <property name="text">
          <string>3.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>184</y>
      <width>144</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>ADC启动：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobutton_adc_yes">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobutton_adc_no">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>144</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>SOC启动：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobutton_soc_yes">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobutton_soc_no">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>161</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="button_start_device">
       <property name="text">
        <string>开启设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_close_device">
       <property name="text">
        <string>关闭设备</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_exit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/logo_hui.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Bodoni MT Black&quot;;</string>
   </property>
   <property name="text">
    <string>Multi-channel acquisition system</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
